docs/content/guides/developer/sui-101/using-events.mdx

```diff
@@ -36,159 +30,220 @@ To create an event in your Move modules, add the `sui::event` dependency.
 use sui::event;
 ```
 
-With the dependency added, you can use the `emit` function to fire an event whenever the action you want to monitor fires. For example, the following code is part of an example application using digital donuts. The `collect_profits` function handles the collection of SUI and emits an event whenever the function is called. To act on that event, you need to subscribe to it.
+With the dependency added, you can use the `emit` function to trigger an event whenever the action you want to monitor fires. For example, the following code is part of an example application that enables the locking of objects. The `lock` function handles the locking of objects and emits an event whenever the function is called.
+
+{@inject: examples/trading/contracts/escrow/sources/lock.move#fun=lock noComments}
+
+## Query events
```

Comment:
Before the "query events" section was connected to the "Emit events in Move" section, because the RPC call was fetching events that were emitted in the previous section, and that's no longer the case. That might be confusing for folks.

Reply:
I saw it as slightly different conceptually but def defer to your judgment on that one.